.github/workflows/macos-universal.yml

```diff
@@ -0,0 +1,207 @@
+name: macOS universal OpenSSL+libssh
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ devel ]
+  pull_request:
+
+env:
+  OPENSSL_VER: "3.1.4"
+  LIBSSH_VER: "0.11.2"
+
+jobs:
+  macos-arm64-build:
+    name: Build (arm64)
+    runs-on: macos-14
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Prep dirs
```

Comment:
Use full words when explaining what steps do. Best avoid generic references to abstract ideas. Be specific.